src/utils/string.ts

```diff
@@ -0,0 +1,6 @@
+export function truncateText(text: string, maxLength: number): string {
```

Comment:
We already have a util file for `strings`: https://github.com/JoinColony/colonyCDapp/blob/a17f72d2d198dd3848865b0910896e2796576a9b/src/utils/strings/index.ts _(which even has a method that can be already used for this)_

Please check before adding more files and util methods in